### DISCUSSION TITLE
Adding fully mature crop indication.

### DISCUSSION THREAD

**Discussion by steelmonkey** [BedrockTweaks/Files#137] [Ideas]
I think it would be nice to add flowers or something to fully grown crops like the "Age 25 Kelp" tweak dose. Make it easier to see which crops are ready for harvest especially wheat, and it would look nice.

**Comment by SquatchHunter** [comment]
this has been proven to be possible and is on the job list, just not merged yet :)

**Comment by drav0011** [comment]
This is being tracked here https://github.com/BedrockTweaks/Files/issues/34
